prime-dotnet-webapi/Migrations/20210614235905_AddSiteStatusHistory.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Prime.Migrations
+{
+    public partial class AddSiteStatusHistory : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Existing data needs to be migrated before column is dropped.
+            // However, new Site rows cannot run into not-nullable constraint.
+            migrationBuilder.AlterColumn<int>(
+                name: "Status",
+                table: "Site",
+                nullable: true);
```

Comment:
Are you intending to add the migration in this PR?

Reply:
Data migration?  It's in https://bcgovmoh.atlassian.net/browse/PRIME-1646

Reply:
What is the status of this SQL? Is it still being tested?
I hope that the intention is to add it to this migration rather than running it against prod directly.

Reply:
Data SQL has been tested locally and was preferred by Jason to run separately from .NET migration.

Reply:
After looking at the migration I dont see any PII data in it. Apologies if I confused the issue.. but if there's no PII it should be run in a migration. Thanks!